src/VlcSource.h

```diff
@@ -50,18 +54,12 @@ class VlcSourceCount final {
     // ACCESSORS
     int lineno() const { return m_lineno; }
     uint64_t count() const { return m_count; }
-    bool ok() const { return m_ok; }
+    bool ok(uint64_t ok_min) const { return m_count >= ok_min; }
 
     // METHODS
-    void incCount(uint64_t count, bool ok) {
-        if (m_count == std::numeric_limits<uint64_t>::max())
-            m_ok = ok;
-        else
-            m_ok = m_ok && ok;
-        m_count = std::min(m_count, count);
-    }
+    void incCount(uint64_t count) { m_count = std::max(m_count, count); }
```

Comment:
Readers will think inc did +=.

```suggestion
    void maxCount(uint64_t count) { m_count = std::max(m_count, count); }
```